schemas/v1/6_gap_find.go

```diff
@@ -0,0 +1,72 @@
+package v1
+
+// Schema version 1 adds gap find function
+
+func init() {
+	patches.Register(
+		6,
+		`
+CREATE FUNCTION {{ .SchemaName | default "public"}}.gap_find(
```

Comment:
can postgres functions be commented? If so it would be helpful to add a short summary describing the arguments this method expects and the results it returns.